README.rst

```diff
@@ -14,15 +14,13 @@ dimod
 
 `dimod` is a shared API for samplers. It provides:
 
-* a binary quadratic model (BQM) class that contains Ising and quadratic
-  unconstrained binary optimization (QUBO) models used by samplers such as the
-  D-Wave system.
-* a discrete quadratic model (DQM) class and higher-order (non-quadratic) models.
+* classes for quadratic models---such as the binary quadratic model (BQM) class that
+  contains Ising and quadratic unconstrained binary optimization (QUBO) models used
```

Comment:
I see what you're doing, but `Ising and quadratic` made me double take a few times reading it. Maybe `Ising and QUBO (quadratic...)` would work better?

Reply:
Done